src/main/java/frc/robot/commands/auto/Autos.java

```diff
@@ -0,0 +1,117 @@
+package frc.robot.commands.auto;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import frc.lib.utils.AllianceFlipUtil;
+import frc.robot.FieldConstants;
+import frc.robot.Orchestrator;
+import frc.robot.subsystems.drive.Drive;
+import org.littletonrobotics.junction.Logger;
+
+public class Autos {
+  private final Drive drive;
+  private final Orchestrator orchestrator;
+  private Translation2d noteTranslation;
+  private Translation2d secondNoteTranslation;
+  private Translation2d thirdNoteTranslation;
+
+  public Autos(Drive drive, Orchestrator orchestrator) {
+    this.drive = drive;
+    this.orchestrator = orchestrator;
+  }
+
+  public enum StartingPosition {
+    LEFT,
+    CENTER,
+    RIGHT
+  }
+
+  private void setNotePositions(StartingPosition position) {
+    switch (position) {
+      case LEFT -> {
+        this.noteTranslation =
+            AllianceFlipUtil.apply(FieldConstants.StagingLocations.spikeTranslations[0]);
```

Comment:
0, 1, 2 would make sense for Blue alliance, but would it be different for Red?

I think 0 is always closest to the Amp, so Right/Left would be flipped.

This logic could go in your getNoteTranslation method (see below) - make sure you comment it well to describe how it works.

Reply:
we are switching the positions based on alliance

Reply:
Won't this still be reversed on the Red alliance, like we talked about yesterday?

LEFT would be spikeTranslations[2] for Red.

See my suggestion about getNoteTranslation.